### PR TITLE
Update tr.json

### PR DESCRIPTION
**Pull request by abdulkadiraktas**

### CODE REVIEW

**Review by tabarra — CHANGES_REQUESTED** [review]
A "hang" or "freeze" is not the same thing as a "crash", that's why I changed the english message.  
And "hang" is not the same thing as a "partial".

**Comment by abdulkadiraktas** [comment]
I see. I fixed.

File: locale/tr.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
     "restarter": {
         "start_timeout": "zamanında başlatılamadı",
         "crash_detected": "Çökme tespit edildi",
-        "hang_detected": "hang detected",
-        "partial_hang_warn": "Due to a partial hang, this server will restart in 1 minute. Please disconnect now.",
-        "partial_hang_warn_discord": "Due to a partial hang, **%{servername}** will restart in 1 minute.",
+        "hang_detected": "Kısmi bir takılma algılandı",
```
Comment (tabarra) [inline locale/tr.json]: Does "Kısmi" mean Partial? If that's the case then its better to remove the work, not to confuse with the partial hang warnings.

Comment (abdulkadiraktas) [reply locale/tr.json]: "hang" has a lot of means from Turkish. In This case "Takılma" is good. I deleted "Kısmi" side.
http://prntscr.com/12q28zd

**Comment by tabarra** [comment]
Thanks! Will merge it soon!

**Pull request merged**